src/ReactiveUI/Platforms/windows-common/ReactiveUserControl.cs

```diff
@@ -100,6 +104,40 @@ class ReactiveUserControl<TViewModel> :
                 typeof(ReactiveUserControl<TViewModel>),
                 new PropertyMetadata(null));
 
+#if HAS_UNO
+#if ANDROID
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ReactiveUserControl{TViewModel}"/> class.
+        /// Native constructor, do not use explicitly.
+        /// </summary>
+        /// <remarks>
+        /// Used by the Xamarin Runtime to materialize native
+        /// objects that may have been collected in the managed world.
+        /// </remarks>
+        /// <param name="javaReference">javaReference.</param>
```

Comment:
Put valid comments in the Params

Reply:
Same as [here](https://github.com/reactiveui/ReactiveUI/pull/2116#discussion_r301410650).